reanahub/reana-server
# emails: use correct k8s namespace and component prefix in templates

Use the correct Kubernetes namespace and component prefix when building
`kubectl` commands in the token request email template.

Closes #615

How to test:
1. Enable notifications and disable email confirmation
```diff
diff --git a/helm/configurations/values-dev.yaml b/helm/configurations/values-dev.yaml
index 41535ee..bff43a2 100644
--- a/helm/configurations/values-dev.yaml
+++ b/helm/configurations/values-dev.yaml
@@ -6,6 +6,7 @@ components:
       environment:
         REANA_SCHEDULER_REQUEUE_SLEEP: 2
         REANA_RATELIMIT_SLOW: "5 per second"
+        REANA_USER_EMAIL_CONFIRMATION: false
     reana_workflow_controller:
       image: docker.io/reanahub/reana-workflow-controller
       environment:
@@ -24,3 +25,9 @@ components:
       image: docker.io/reanahub/reana-message-broker
     reana_ui:
       image: docker.io/reanahub/reana-ui
+
+notifications:
+  enabled: true
+  email_config:
+    sender: sender@a.it
+    receiver: receiver@b.it
```
2. Deploy in custom namespace with custom name
`reana-dev cluster-deploy --admin-email john.doe@example.org --admin-password mysecretpassword --mode debug --instance-name xyz --namespace ns`
3. Create a new user
4. Request new token
5. Check the email that is sent from the web UI at http://localhost:32580
